src/components/figures/chart/chartJs/chartjs_show_values_plugin.ts

```diff
@@ -36,7 +36,7 @@ export const chartShowValuesPlugin: Plugin = {
     ctx.save();
     const { left, top, height, width } = chart.chartArea;
     ctx.beginPath();
-    ctx.rect(left, top, width, height);
+    ctx.rect(left - 10, top - 10, width + 20, height + 20);
```

Comment:
Yeah I noticed that the clip kinda sucks ... but 10px seems arbitrary, we still can have things cut...